common/helpers/move/get-locals.js

```diff
@@ -21,7 +21,17 @@ function getLocals(req) {
   const { person } = profile
 
   // person
-  const personalDetailsSummary = presenters.personToSummaryListComponent(person)
+  // NB. category for move is actually on the move's profile
+  // since it's a snapshot rather than the current status
+  const personWithProfileCategory = person
```

Comment:
There's not a "person without profile category" is there?

Reply:
The var name is to make it explicit that the person object is having the profile category merged into it. This is essentially a wrinkle of the person/profile divide. Discussed with Dom as the better (for now anyway) route of handling this rather than either a) mixing in the profile category during the transformations or b) passing profile as an extra param to the presenter.

Reply:
Thanks for the explanation :smile:

Reply:
Not one of now, but I'm wondering if the move page should actually start doing everything off a `profile`, which should always have a `person` relationship and we actually pass the profile into the presenters to build things like the personal details summary, right hand panel, etc.

Reply:
Have made the comment a bit more verbose and in line with what I just said